.travis.yml

```diff
@@ -65,10 +65,14 @@ matrix:
         - OS_TYPE=centos_docker
         - OS_VERSION=7
         - EXTRA_SSH=-oHostKeyAlgorithms=+ssh-dss
+    - os: linux
+      dist: bionic
+      env:
+        - EXTRA_SSH=-oHostKeyAlgorithms=+ssh-dss
```

Comment:
nope; Travis-CI now offers bionic images via `dist: bionic`
https://docs.travis-ci.com/user/reference/bionic/
[maybe i''ll reference this link in the commit]

`OS_TYPE=ubuntu_docker` is needed so that we go directly to https://hub.docker.com/_/ubuntu  and get what we need from there;
Travis-CI is really sluggish at providing really-new Ubuntu images;
but they do provide docker integration to use other images which they don't support [directly]

so, using whatever is on dockerhub gets us faster/closer to newer Ubunut versions by up-to 2-3 years [sounds like a commercial]

on a related note, we can also add CentOS 8; https://hub.docker.com/_/centos
i don't think i will [in the short future]; so it's free for others to try

my main interest [right now] in Ubuntu 20.04 is GNU-radio 3.8;
with this version already packaged into Ubuntu 20.04 we can avoid having to build gnuradio in gr-iio and scopy;
which means that we can make the scopy builds green again

Reply:
so
I found out that CentOS 8 has GNUradio 3.8
https://centos.pkgs.org/8/epel-x86_64/gnuradio-3.8.0.0-5.el8.x86_64.rpm.html

i might consider adding it to libiio sooner than later;